From 41b34d6d678f3872d13d048820a94c208196e33e Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Tue, 7 May 2019 14:27:53 +0200
Subject: [PATCH 6/7] arm64: dt: mb-smarc-2-common: add eeprom 24c32

this eeprom is primary intended to be used for EDID data @ x86 but can be
used free for ARM

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm64/boot/dts/freescale/mb-smarc-2-common.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/mb-smarc-2-common.dtsi b/arch/arm64/boot/dts/freescale/mb-smarc-2-common.dtsi
index e107ce0f096c..809f262794f1 100644
--- a/arch/arm64/boot/dts/freescale/mb-smarc-2-common.dtsi
+++ b/arch/arm64/boot/dts/freescale/mb-smarc-2-common.dtsi
@@ -49,6 +49,15 @@
 	};
 };
 
+&i2c1 {
+	eeprom2: at24c32@57 {
+		compatible = "st,24c32", "atmel,24c32", "at24";
+		reg = <0x57>;
+		pagesize = <32>;
+		status = "okay";
+	};
+};
+
 &iomuxc {
 	tqma8xxs-mb-smarc-2 {
 		pinctrl_lpuart0: lpuart0grp {
